src/github/queries.gql

```diff
@@ -836,7 +836,20 @@ query MaxIssue($owner: String!, $name: String!) {
 	}
 }
 
-query GetMilestones($owner: String!, $name: String!, $assignee: String!) {
+query GetMilestones($owner: String!, $name: String!, $states: [MilestoneState!]!) {
+	repository(owner: $owner, name: $name) {
+			milestones(first: 12, orderBy: { direction: DESC, field: DUE_DATE }, states: $states) {
```

Comment:
What if a repo has more than 12 open milestones? This might not be a good habit but I'm sure there will be users who do have more. Also, in the case that both open and closed Milestones want to be fetched, it will be common that there are more than 12 milestones (vscode has almost 200).

Reply:
Fair point. I'll up to 100 in another PR. If we need to add paging we can always add it later.